platformio.ini

```diff
@@ -4,6 +4,14 @@ src_dir = src
 extra_configs = platformio_extra.ini
 
 [env]
+platform = espressif32 @^5
+board = az-delivery-devkit-v4
+board_build.filesystem = littlefs
+board_build.partitions = huge_app.csv
```

Comment:
I think huge_app doesn't allow ota anymore

Reply:
Compared to https://github.com/espressif/arduino-esp32/blob/master/tools/partitions/no_ota.csv, the code partition is a bit smaller and spiffs is larger

Reply:
I've more or less maximized the filesystem in PR #326 already

Reply:
default.csv then, partitions_4M.csv is a partition scheme part of Espressif IDF not the Arduino framework.

Reply:
It's a file that's in the PR

Reply:
Either way, it works fine with default.csv, on PR #326 as well.